packages/components/src/components/editor/browser/browser.tsx

```diff
@@ -63,6 +63,11 @@ export const useBrowser = (
   ];
 
   const [filter, setFilter] = React.useState(options?.initialSearch ?? '');
+  const globalFilterFn = (row: Row<BrowserNode>, _columnId: string, filterValue: string) => {
+    filterValue = filterValue.toLowerCase();
```

Comment:
It is noramlly better to not change a variable which is provided by a parameter, better create a new one.